src/App.js

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Buttons from "./components/Buttons";
+import Form from "./components/Form";
+import logo from "./assets/screenshot-logo.png";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    //set functions for the fruits
+    const [aardbei, setAardbei] = useState(0);
+    const [banaan, setBanaan] = useState(0);
+    const [apple, setApple] = useState(0);
+    const [kiwi, setKiwi] = useState(0);
+    const order = `aardbei: ${aardbei} banaan: ${banaan} appel: ${apple} kiwi: ${kiwi}` ;
+    console.log(order)
+
+
+  //  reset function
+  function reset (){
+      setBanaan(0);
+      setApple(0);
+      setAardbei(0);
+      setKiwi(0);
+  }
+    return (
+        <>
+            <img src={logo} alt="photo"/>
+            <Buttons name="🍓 Aardbei" fruit={aardbei} setFruit={setAardbei}/>
+            <Buttons name="🍌 Banaan" fruit={banaan} setFruit={setBanaan}/>
+            <Buttons name="🍎 Appels" fruit={apple} setFruit={setApple}/>
+            <Buttons name="🥝 Kiwi's" fruit={kiwi} setFruit={setKiwi}/>
+            <button className="reset" onClick={reset}>Reset</button>
+            <Form />
+        </>
+    );
```

Comment:
Als je de waardes van het fruit wil afdrukken wanneer je op de button klikt, dan zul je in deze file nog een functie te maken die die waardes in de console afdrukt. Deze functie kun je dan meegeven aan het Form-component en uitvoeren wanneer op de knop gedrukt wordt.
```suggestion
    function onSubmit() {
        console.log(order);
    }

    return (
        <>
            <img src={logo} alt="photo"/>
            <Buttons name="🍓 Aardbei" fruit={aardbei} setFruit={setAardbei}/>
            <Buttons name="🍌 Banaan" fruit={banaan} setFruit={setBanaan}/>
            <Buttons name="🍎 Appels" fruit={apple} setFruit={setApple}/>
            <Buttons name="🥝 Kiwi's" fruit={kiwi} setFruit={setKiwi}/>
            <button className="reset" onClick={reset} onSubmit={onSubmit}>Reset</button>
            <Form />
        </>
    );
```